scratch-js/Sprite.mjs

```diff
@@ -206,12 +206,10 @@ class SpriteBase {
     this._project.restartTimer();
   }
 
-  startSound(soundName) {
+  *startSound(soundName) {
```

Comment:
Is `sb-edit` going to have to change now that this is a generator?

Reply:
Oh, yep - I have it changed locally but forgot to make a PR for that. Thanks!